windows/win_scheduled_task.py

```diff
@@ -32,20 +29,45 @@
   name:
     description:
       - Name of the scheduled task
-      - Supports * as wildcard
     required: true
   enabled:
```

Comment:
Confused by this comment, name is a required field.
`$name = Get-Attr -obj $params -name name -failifempty $true -resultobj $result